guides/doc-Provisioning_Guide/topics/Resources.adoc

```diff
@@ -330,6 +340,7 @@ To set a default encrypted password for your hosts, complete the following steps
 # python -c 'import crypt,getpass;pw=getpass.getpass(); print(crypt.crypt(pw)) if (pw==getpass.getpass("Confirm: ")) else exit()'
 -----------------
 +
+Alternatively you can use pwkickstart command from distribution repositories or EPEL.
```

Comment:
What is happening here? Looks like it should be wrapped in an if statement? 
Otherwise, it would be logged as an issue and not included in this PR. This PR should only be for Foreman/Satellite issues.

Reply:
My bad, tried to squeeze in this improvement. Can we move this into a different PR please?